Automattic/wp-calypso
# WPCOM: Migrate `wpcom.undocumented()` keyrings to `wpcom.req`

#### Changes proposed in this Pull Request

This PR migrates the `wpcom.undocumented()` methods for site keyrings to `wpcom.req.get()`.

Part of the ongoing effort to get rid of `wpcom.undocumented()`.

#### Testing instructions
* Go to `/marketing/connections/:site` where `:site` is one of your sites.
* Verify current keyrings are still retrieved correctly (currently, only used by Google My Business).
* Verify you can still successfully connect Google My Business.
* Verify you can still successfully connect and disconnect a Google My Business location (AKA listing).
* Verify you can still successfully disconnect a Google My Business connection.
